htdocs/router.php

```diff
@@ -0,0 +1,90 @@
+<?php
+error_log("URL:  " . $_SERVER['REQUEST_URI']);
+if (preg_match('#^/bootstrap/(.*)#', $_SERVER["REQUEST_URI"])){
+	// Bootstrap is not a Loris module, don't rewrite it.
+	// RewriteRule ^bootstrap/(.*) bootstrap/$1 [L]
+	return false;
+} else if (preg_match('#^/([0-9]{6,6})/$#', $_SERVER["REQUEST_URI"])) {
+	// Redirect /CandID/ to visit label list
+ 	// RewriteRule ^([0-9]{6,6})/$ /main.php?test_name=timepoint_list&candID=$1
+ 	$getParams = explode("/", $_SERVER["REQUEST_URI"]);
+	$_REQUEST["test_name"] = "timepoint_list";
+	$_REQUEST['candID'] = $getParams[1];
```

Comment:
I think that, in general, you need to set both $_REQUEST and $_GET ($_REQUEST is the union of $_GET and $_POST in PHP).. you should probably make a helper function to do that and just call that.